README.md

```diff
@@ -175,8 +174,12 @@ In order to stream audio, you will need to setup a streaming client and a media
 ```javascript
 import { RevAiStreamingClient } from 'revai-node-sdk';
 
-const audioConfig = new AudioConfig() // Initialize audio configuration for the streaming client
-const streamingClient = new RevAiStreamingClient("ACCESS TOKEN", audioConfig);
+ // Initialize audio configuration for the streaming client
+const audioConfig = new AudioConfig()
+
+// Optionally set the specific Rev AI deployment of your account, defaults to the US deployment.
+// Learn more about Rev AI's global deployments at https://docs.rev.ai/api/global-deployments.
+const streamingClient = new RevAiStreamingClient({ token: "ACCESS TOKEN", deploymentConfig: RevAiAPiDeploymentConfigMap.get(RevAiAPiDeployment.US) }, audioConfig);
```

Comment:
```suggestion
const streamingClient = new RevAiStreamingClient({ token: "<ACCESS_TOKEN>", deploymentConfig: RevAiAPiDeploymentConfigMap.get(RevAiAPiDeployment.US) }, audioConfig);
```

Please replace instances of `ACCESS TOKEN` with `<ACCESS_TOKEN>` as it is our convention.